src/programs.rkt

```diff
@@ -378,7 +378,8 @@
             (and (andmap identity args*) (cons op args*)))
           (let ([op* (apply get-parametric-operator 
                             (hash-ref parametric-operators-reverse op)
-                            (make-list (length args) prec*))]
+                            (make-list (length args) prec*)
+                            #:fail-fast? #f)]
```

Comment:
Can you give a quick summary of this change? Is it a bug fix?

Reply:
Bug fix